phpBB/phpbb/captcha/plugins/qa.php

```diff
@@ -100,6 +100,28 @@ function init($type)
 			$db->sql_freeresult($result);
 		}
 
+		// final fallback to any language
+		if (!sizeof($this->question_ids))
+		{
+			$this->question_lang = '';
+
+			$sql = 'SELECT q.question_id, q.lang_iso
+				FROM ' . $this->table_captcha_questions . ' q, ' . $this->table_captcha_answers . ' a
+				WHERE q.question_id = a.question_id
+				GROUP BY lang_iso';
```

Comment:
Because it is now incorrect. 
mysql searching through both tables for this column whether exists or not.